src/routes/[search=searchType].svelte

```diff
@@ -78,6 +81,28 @@
 
 	let previousParams = { ...searchParams }
 
+	let modLoaders = loaders.filter((thing) => {
+		// return thing.supported_project_types.includes("mod") &&
+		return thing.name.includes("fabric") ||
+		thing.name.includes("forge") ||
+		thing.name.includes("quilt") ||
+		thing.name.includes("liteloader") ||
+		thing.name.includes("modloader") ||
+		thing.name.includes("rift")
+	})
+
+	let pluginLoaders = loaders.filter((thing) => {
+		return thing.name.includes("purpur") ||
+		thing.name.includes("bukkit") ||
+		thing.name.includes("paper") ||
+		thing.name.includes("spigot") ||
+		thing.name.includes("sponge")
+	})
+
+	// let pluginLoaders = loaders.filter((type) => type.name.includes("purpur" || "bukkit"))
+
+	console.log(loaders)
+
```

Comment:
This could probably be made more concise, likely also by adjustments in the generated files.